### PR TITLE
KDE Gear 23.08.1

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**

Updates KDE Gear Applications to 23.08.1

`kalendar` has been renamed to `merkuro` for this release

`kirigami-addons` has also been updated to 0.11.0

**Test Plan**

Use a few applications as normal

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/m/merkuro/package.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - kdepim-runtime
+    - kirigami-addons
+setup      : |
+    %cmake_ninja
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
+replaces   :
+    - kalendar
```
Comment (ReillyBrogan) [inline packages/m/merkuro/package.yml:54]: Please add a commit deleting Kalendar from the repo

File: packages/m/merkuro/package.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - dbginfo : kalendar-dbginfo
```
Comment (ReillyBrogan) [inline packages/m/merkuro/package.yml:55]: Please also add `kalendar` and `kalendar-dbginfo` to the solus-sc deprecations

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]

**Review by ReillyBrogan — APPROVED** [review]
LGTM, feel free to land

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
I did a bunch of rebuilds of KDE Gear applications for my updates to gmime, gpgme, gnupg, and qtkeychain. This needs a rebase unfortunately

**Comment by joebonrichie** [comment]
Reminder to self: Update ktextaddons and do it's rebuilds after this lands.

**Review by ReillyBrogan — DISMISSED** [review]

**Comment by joebonrichie** [comment]
Merged manually.

**Pull request closed without merging**